UnityProject/Assets/Scripts/Core/Transform/Orientation.cs

```diff
@@ -1,284 +1,286 @@
 using System;
-using System.Collections.Generic;
 using UnityEngine;
 
-/// <summary>
-/// Represents an absolute facing. Can only be in 4 cardinal directions.
-///
-/// Defined in terms of euler angle (rotation about the z axis, right is 0 and up is 90).
-/// </summary>
-public struct Orientation : IEquatable<Orientation>
+namespace Core.UsTransform
```

Comment:
There's `Core.Directionals` already in use, maybe slap it in there?

Reply:
I will look into it after work. It sounds a lot better tho, at least for this enum in particular. Does it apply for the rest of files in the folder?

Reply:
Nah, not really. Hmm, I think it's fine to deviate from the folder name tbh. Well, up to you.

Reply:
Could do `Core.Transforms`?

Reply:
That might work

Reply:
done

Reply:
Hmm, what's done? : )

Reply:
lmao I never pushed